PendingReleaseNotes

```diff
@@ -1,6 +1,8 @@
 >= 14.2.11
 ----------
 
+* The ``ceph df`` command now lists the number of pgs in each pool.
```

Comment:
this should be under `>= 14.2.12`

Reply:
May I go ahead and drop all notes which are under `14.2.11` as it the `14.2.11` is already out? and keep this note in `14.2.12`.

Reply:
hmm, looks like the one about `OSD_TOO_MANY_REPAIRS` should also move to 14.2.12. @smithfarm any preferences of how to handle these?

Reply:
@neha-ojha The general idea is to examine the previous release notes ("Notable changes" section) - https://ceph.io/releases/v14-2-11-nautilus-released/#notable-changes in this case - and delete only those notes that were actually published. After completing this step, any remaining notes from v14.2.11 should be considered "not yet published" and dealt with on a case-by-case basis. Unfortunately, it's not always immediately obvious to which backport PR each release note relates.

Reply:
This one - https://github.com/ceph/ceph/pull/36944/files#diff-91e5380318f1e66a0b2ad947685cb4caL16-R25 is not yet published and the other two are.  The one which is not published is from the monitor component for option ``mon_osd_warn_num_repaired``. Please let me know how you guys want to handle this one other two which are already published I can remove those.

Reply:
I vote to:

* change heading ">= 14.2.11" to ">= 14.2.12"
* remove all bullet points that have already been published

Bullet points that have not yet been published should stay.

Reply:
Thanks. I have made the changes.